app/src/db/migrations/20200529145912_008-settings-request-access.js

```diff
@@ -0,0 +1,28 @@
+const PREFIX = require('../../forms/minesoperatorscreening/constants').PREFIX;
+
+exports.up = function(knex) {
+  return Promise.resolve()
+    .then(() => {
+      const data = {
+        name: 'accessRequestedEmail',
+        enabled: true,
+        config: {
+          template: 'access-requested-email.html',
+          from: 'MCAD.RegionalOps@gov.bc.ca',
+          to: 'NR.CommonServiceShowcase@gov.bc.ca',
+          subject: 'Mines Operator Screening - Access Requested',
+          title: 'Access Requested',
+          priority: 'normal',
+          message: 'A request for access has been received for your Mines Operator Screening administration page.',
```

Comment:
Would consider adding the name of the requestor in or some identifier?
Otherwise if 10 people request access this is just 10 of the identical email showing up in the inbox.

Reply:
adding in, good idea